pkg/logging/logger.go

```diff
@@ -86,10 +94,16 @@ func GetLogger(logLevel, encoding, logDir string, encodingLevel func(zapcore.Lev
 	}
 
 	if logDir != "" {
-		zapConfig.OutputPaths = append(zapConfig.OutputPaths, filepath.Join(logDir, logFileName))
+		logDirPath := filepath.Join(logDir, logFileName)
+		// currently zap's default file sink do not support handling of windows files
+		// so if we are on windows os register an sink which will handle the opening of file.
+		if utils.IsWindowsPlatform() {
+			zap.RegisterSink(terrascanZapSink, newTerrascanWinFileSink)
+			logDirPath = terrascanZapSinkWithSeparator + logDirPath
+		}
+		zapConfig.OutputPaths = append(zapConfig.OutputPaths, logDirPath)
 	}
-
-	// create zap logger
+	// should we panic here?
```

Comment:
I think we should panic if we cannot create logger, or, propagate error back and exit with error message.